articles/load-balancer/load-balancer-troubleshoot.md

```diff
@@ -65,9 +65,9 @@ To verify, you can conduct a network trace. The Fully Qualified Domain Name (FQD
 
 If you decide to open a support case, collect the following information for a quicker resolution. Choose a single backend VM to perform the following tests:
 
-- Use `ps ping` from one of the backend VMs within the virtual network to test the probe port response (example: ps ping 10.0.0.4:3389) and record results. 
+- Use `PsPing` from one of the backend VMs within the virtual network to test the probe port response (example: ps ping 10.0.0.4:3389) and record results. 
```

Comment:
Thanks for the update.
Please change to the following (Note `psping` added to example):
Use `PsPing` from one of the backend VMs within the virtual network to test the probe port response (example: psping 10.0.0.4:3389) and record results.